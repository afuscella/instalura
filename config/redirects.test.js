/* eslint-disable no-console */
import redirects from './redirects';

console.log(redirects);
describe('config/redirects', () => {
  test('when redirect is requested should match static def', () => {
    const redirectsExpected = redirects;
    const redirectsActual = [{
      source: '/login/',
      destination: '/app/login/',
      permanent: true,
    }];
    // @Assert
    expect(redirectsExpected).toEqual(redirectsActual);
  });

  test('when redirect is requrest should match the snapshot', () => {
    const redirectsExpected = redirects;

    // @Assert
    expect(redirectsExpected).toMatchSnapshot();
  });
});
