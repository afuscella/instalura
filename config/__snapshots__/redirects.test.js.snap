// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config/redirects when redirect is requrest should match the snapshot 1`] = `
Array [
  Object {
    "destination": "/app/login/",
    "permanent": true,
    "source": "/login/",
  },
]
`;
